admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -76,6 +76,8 @@ Plan very carefully before enabling encryption, because it is not reversible via
 If you lose your encryption keys, your files are *not* recoverable. 
 Always have backups of your encryption keys stored in a safe location, and consider enabling all recovery options.
 
+.. warning:: Consider there are 2 different encryptions and they are **not compatible**. There is Basic encryption and Master Key Based encryption. 
```

Comment:
Nor do most of the words in the docu -.- I just try to fit in the writing style :)